cli/tests/unit/truncate_test.ts

```diff
@@ -12,6 +12,20 @@ unitTest(
     assertEquals(Deno.readFileSync(filename).byteLength, 5);
     Deno.truncateSync(filename, -5);
     assertEquals(Deno.readFileSync(filename).byteLength, 0);
+
+    const file = Deno.openSync(filename, {
+      read: true,
+      write: true,
+    });
+
+    Deno.truncateSync(file.rid, 20);
+    assertEquals(Deno.readFileSync(filename).byteLength, 20);
+    Deno.truncateSync(file.rid, 5);
+    assertEquals(Deno.readFileSync(filename).byteLength, 5);
+    Deno.truncateSync(file.rid, -5);
+    assertEquals(Deno.readFileSync(filename).byteLength, 0);
+
+    Deno.close(file.rid);
```

Comment:
For these unit tests, can you create a new test case, rather than adding to existing ones?